apps/passport-client/components/modals/Modal.tsx

```diff
@@ -7,18 +7,38 @@ import { Spacer } from "../core";
 import { CircleButton } from "../core/Button";
 import { icons } from "../icons";
 import { InfoModal } from "./InfoModal";
+import { InvalidParticipantModal } from "./InvalidParticipantModal";
 import { SaveSyncModal } from "./SaveSyncModal";
 import { SettingsModal } from "./SettingsModal";
 
 export function MaybeModal() {
   const [state, dispatch] = useContext(DispatchContext);
   const close = useCallback(
-    () => dispatch({ type: "set-modal", modal: "" }),
+    () =>
+      dispatch({ type: "set-modal", modal: "", modalUnDismissable: undefined }),
```

Comment:
nit, you may be able to just have

```ts
function isUndismissable(modal: ZuState['modal']) {
  switch(modal) {
    case "invalid-participant":
    case "save-sync":
      return true;
    default:
      return false;
  }
}
```

..local to `Modal.tsx` and not have to add anything to global state